components/brave_rewards/resources/page/components/ads_panel.tsx

```diff
@@ -272,12 +272,15 @@ export function AdsPanel () {
     return (
       <>
         <style.paymentStatus>
-          <PaymentStatusView
-            minEarnings={adsData.adsMinEarningsLastMonth}
-            maxEarnings={adsData.adsMaxEarningsLastMonth}
-            nextPaymentDate={adsData.adsNextPaymentDate}
-            providerPayoutStatus={providerPayoutStatus}
-          />
+          {
+            externalWallet &&
```

Comment:
I think this check is not enough. If the user only clicked on one of the wallet providers in the connect wallet modal, but didn't actually go through the linkage, they'll have an external wallet.

Reply:
Or do we make sure everywhere that it's null on `kNotConnected`?

Reply:
We should change the external wallet's representation in `Preferences`, then it would be trivial, as `kNotConnected` wouldn't even exist. 🙂